src/InvalidArgumentExceptionInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Dhii\Exception;
+
+/**
+ * Represents an exception that signals an invalid function argument.
+ *
+ * @since [*next-version*]
+ */
+interface InvalidArgumentExceptionInterface extends ThrowableInterface
```

Comment:
Perhaps it would also be useful for the exception to provide the argument name or position. What do you think?